frontend/src/Requirements.js

```diff
@@ -15,8 +15,9 @@ export function toggleSettle(course, path_to, settle){
   let courseOnScheduleNotAssigned = '';
   // checks for duplicates on course schedule and grabs courses that do not have the path and courses that do have the path
   allCoursesOnSchedule.each(function(){
-    if($(this).data('reqs').indexOf(path_to) === -1)
-      courseOnScheduleNotAssigned = $(this);
+    if($(this).data('reqs').map((path)=>{
+      return '//'.join(path.split('//')[:3])
```

Comment:
This gives me a compile error:
![image](https://user-images.githubusercontent.com/9757500/42671319-68abd352-862d-11e8-9877-80133d5bddc7.png)